docs/howto/promote-css-changes.md

```diff
@@ -0,0 +1,80 @@
+# How to promote CSS changes to Live with minimal pain
+
+This document describes the best workflow for promoting changes to CSS styles
+to the Live tier in Pantheon, in such a way that they take effect relatively
+quickly and without confusion about whether things worked.
+
+## Why this is necessary
+
+Promoting changes to CSS stylesheets can be a tricky proposition, because the
+files involved can be cached in many different places. Additionally, many of
+our stylesheets are compiled via SCSS - but this does not happen on every page
+load on the Test and Live tiers, because it raises performance problems.
+
+As a result, there can at times be a significant delay between deploying a new
+set of styles and seeing those changes reflected in users' browsers.
+
+Following the steps below may not eliminate that delay entirely, but should help
+reduce it.
+
+## When to follow these steps
+
+Follow this procedure whenever a change in being promoted on Pantheon which
+affects any stylesheet in the themes.
+
+These steps should be followed when making the change to either the Live or Test
+tiers.
+
+## The procedure
```

Comment:
This procedure has a lot of force reloading that end users won't be doing. While I agree it results in a developer's browser having the correct version of styles, I'm not (yet) convinced I have a clear understanding of if this results in _everyone_ getting the expected styles.

I'd like to leave this PR open for a bit so we can do some cross individual testing... i.e. if one of us does this, does the other person see the expected styles without force reloading, etc.